packages/ddex/processor/src/db.ts

```diff
@@ -130,6 +130,14 @@ export const userRepo = {
     dbUpsert('users', user)
   },
 
+  updateField(id: string, field: keyof UserRow, value: string | Date) {
+    return db.run(sql`
+      update users
+      set ${field} = ${value}
```

Comment:
the `${field}` interpolation here will be converted by the `sql` tagged template to a bind using `?`...

Instead I'd remove this and at the call site use `upsert` (which will only touch the `handle` field and leave other data intact):

```
userRepo.upsert({
  id: user.id,
  handle: resp.handle
})
```

Reply:
added [here](https://github.com/AudiusProject/audius-protocol/pull/8380/commits/2c26e0cb860c0a05bfbf6e032a6a89ecddb370c0). also meant to update the name, not the handle, which is immutable. fixed in that commit